lte/gateway/c/oai/include/messages_def.h

```diff
@@ -37,11 +37,15 @@
 #include "ip_forward_messages_def.h"
 #include "s11_messages_def.h"
 #include "s1ap_messages_def.h"
+#include "ngap_messages_def.h"
 #include "s6a_messages_def.h"
 #include "sctp_messages_def.h"
 #include "gx_messages_def.h"
 #include "mme_app_messages_def.h"
+#include "amf_app_messages_def.h"
 #include "service303_messages_def.h"
 #include "sgs_messages_def.h"
 #include "async_system_messages_def.h"
 #include "udp_messages_def.h"
+#include "ha_messages_def.h"
```

Comment:
why is this on this PR?

Reply:
Removed

Reply:
Hi Ulas some base version issue crept in the PR, if I am removing #include "ha_messages_def.h", code is not compiling.